source/demo/index.js

```diff
@@ -12,6 +12,4 @@ import Application from './Application';
 render(<Application />, document.getElementById('root'));
 
 // Import and attach the favicon
-document.querySelector(
-  '[rel="shortcut icon"]',
-).href = require('file!./favicon.png');
+document.querySelector('[rel="shortcut icon"]').href = require('./favicon.png');
```

Comment:
This change (removing the `file!` prefix) breaks the loader for the favicon.
<img width="876" alt="screen shot 2017-11-26 at 8 59 58 am" src="https://user-images.githubusercontent.com/29597/33242340-336da7d2-d288-11e7-91d6-1ab6e6e93d3f.png">

I'm curious, was this an accident?

Reply:
Yes, sorry. I had problems with autoreloading in webpack 1, so I changed webpack configuration to use webpack 3. This line left in the PR was an accident. I'll get it back in the next commit